draft-ietf-rats-corim.md

```diff
@@ -775,45 +775,54 @@ configuration files, read-only memory, fuses, IO ring configuration, partial
 reconfiguration regions, etc. Measurements comprise raw values, digests, or
 status information.
 
-An environment has one or more measurable elements. Each element can have a
-dedicated measurement or multiple elements could be combined into a single
-measurement. Measurements can have class, instance or group scope.  This is
-typically determined by the triple's environment.
+An environment has one or more measured elements.
+A measured element can have a dedicated measurement, such as a security version number.
+Multiple elements could be combined into a single measurement, such as a running hash of multiple files.
+A measured element can also have several measurement values, such as a name, version, and digest.
+Measurements can have class, instance or group scope.
+A `measurement-map`'s scope is determined by the `environment-map` it is paired with.
 
 Class measurements apply generally to all the Attesters in the given class.
 Instance measurements apply to a specific Attester instance.  Environments
 identified by a class identifier have measurements that are common to the
 class. Environments identified by an instance identifier have measurements that
 are specific to that instance.
 
-The supply chain entity that is responsible for providing the the measurements (i.e. Reference Values or Endorsed Values)
-is by default the CoRIM signer. If a different entity is authorized to provide measurement values,
-the `authorized-by` statement can be supplied in the `measurement-map`.
-
+The supply chain entity that is defined as the authority responsible for providing the the measurements (i.e. Reference Values or Endorsed Values)
+is by default the CoRIM signer.
+If different authorities may provide measurement values, the `"__authorities"` statement can be supplied in the `measurement-map`.
+If the `"__authorities"` statement is used within a condition, at least one of the listed authorities must match the respective ACS entry for the statement to match.
 
 ~~~ cddl
 {::include cddl/measurement-map.cddl}
 ~~~
 
+
 The following describes each member of the `measurement-map`:
```

Comment:
```suggestion
The `measurement-map` contains one or more measured-entities, each is identified by the map key.
It may also contain special values which apply to all measured-entities.
```

Reply:
```suggestion
The `measurement-map` contains one or more measured elements, each is identified by the map key.
```

> It may also contain special values which apply to all measured-entities.

Should this be the negative value space?
See "fixme-authorities" for context

Reply:
I agree with Henk that L802 is insufficient.  It needs some clear description of the negative codepoint space which is reserved for common attributes.